jenkins/bootstrap_test.py

```diff
@@ -1262,6 +1263,25 @@ def Check(job, name):
             'job-configs/kubernetes-jenkins-pull/bootstrap-pull.yaml',
             Check, use_json=is_modern)
 
+    def testBootstrapSecurityPullYaml(self):
+        is_modern = lambda name: 'unit' in name or 'verify' in name
```

Comment:
This is also changed recently - https://github.com/kubernetes/test-infra/blob/master/jenkins/bootstrap_test.py#L1246-L1247